packages/core/src/App/Components/Elements/NotificationMessage/notification.jsx

```diff
@@ -43,8 +43,9 @@ const Notification = ({ data, removeNotificationMessage }) => {
                     onClose={destroy}
                 />
             );
-        case 'trustpilot':
-            return (
+        case 'trustpilot': {
+            // Trustpilot notification will be displayed only after a week of account creation
+            return config?.account_open_since_days >= 7 ? (
```

Comment:
@likhith-deriv Can you prevent notification from being triggered instead of checking condition here?